aws-datastore/src/test/java/com/amplifyframework/datastore/syncengine/OrchestratorTest.java

```diff
@@ -70,19 +73,18 @@ public void itemsPlacedInStorageArePublishedToNetwork() throws AmplifyException
             .name("Susan Quimby")
             .build();
 
-        HubAccumulator accumulator =
-            HubAccumulator.create(HubChannel.DATASTORE, publicationOf(susan), 1)
+        HubAccumulator orchestratorInitObserver =
+            HubAccumulator.create(HubChannel.DATASTORE, DataStoreChannelEventName.READY, 1)
                 .start();
 
-        // Mock behaviors from AppSync
-        AppSync appSync = mock(AppSync.class);
-        ModelMetadata metadata = new ModelMetadata(susan.getId(), false, 1, Temporal.Timestamp.now());
-        ModelWithMetadata<BlogOwner> modelWithMetadata = new ModelWithMetadata<>(susan, metadata);
-        AppSyncMocking.create(appSync).mockResponse(susan);
-        AppSyncMocking.sync(appSync).mockSuccessResponse(BlogOwner.class, modelWithMetadata);
-        AppSyncMocking.onCreate(appSync).callOnStart();
-        AppSyncMocking.onDelete(appSync).callOnStart();
-        AppSyncMocking.onUpdate(appSync).callOnStart();
+        // Mock behaviors from for the API category
+        GraphQLBehavior mockApi = mock(GraphQLBehavior.class);
+
+        ApiMocking.mockSubscriptionStart(mockApi);
```

Comment:
I forget exactly why I went with the static methods in the `AppSyncMocking` util. I think my goal was to get something that looked like Mockito:
```java
onCreate(appSync).callOnStart();
```

In hind-sight, it might have been a better design to create an object instance that encapsulated the `appSync` instance.

Or, in other words, what if your utility worked like:
```java
ApiMocking mocking = new ApiMocking(mock(GraphQLBehavior.class));
```
Or even  just:
```java
// internally, encapsulates a `Mockito.mock(GraphQLBehavior.class)`
MockGraphQLApi api = new MockGraphQLApi();
```

Reply:
Let me circle back on this one. Let me know what you think of the other changes.